packages/react/src/index.ts

```diff
@@ -1,4 +1,27 @@
+import { addGlobalEventProcessor, SDK_VERSION } from '@sentry/browser';
```

Comment:
That we import `SDK_VERSION` here is not optimal.
The versions should never go out of sync but maybe it's better to also create a `version.ts` file like in the other packages and put it there. (Name & Version).